applications/nrf_desktop/board_configuration.rst

```diff
@@ -178,3 +178,9 @@ Sample mouse or dongle (``nrf54h20dk/nrf54h20/cpuapp``)
         For detailed information on working with the nRF54H20 DK, see the :ref:`ug_nrf54h20_gs` documentation.
       * The configurations use the Software Updates for Internet of Things (SUIT) and support firmware updates using the :ref:`nrf_desktop_dfu`.
         Configurations acting as HID peripherals also support firmware updates using the :ref:`nrf_desktop_dfu_mcumgr`.
+
+      .. note::
+         The nRF Desktop application does not build or run for the ``nrf54h20dk/nrf54h20/cpuapp`` board target due to the IronSide SE migration.
```

Comment:
Consider adding that it is only for release 3.1 - I know that this note is only for this release but then I know that it is not a leftover

Reply:
Also in other places

Reply:
then we will add extra release


IMO there s higher chance we will forget to remove it then that it will not be fixed till 3.2

Reply:
What about the NCS 3.1.1 and NCS 3.1.2 releases?

There is also high chance that Fast Pair samples won't get fixed for the next NCS, as it depends on some build system work.

Reply:
ok, as you wish